www/docs/api/key.php

```diff
@@ -62,31 +72,44 @@
 
 # ---
 
-function create_key($commercial, $reason) {
+function create_key($commercial, $reason, $estimated_usage) {
     global $THEUSER;
     $key = auth_ab64_encode(urandom_bytes(16));
     $db = new ParlDB;
     if ($commercial=='') $commercial = 0;
-    $db->query('INSERT INTO api_key (user_id, api_key, commercial, created, reason) VALUES
-        (:user_id, :key, :commercial, NOW(), :reason)', array(
+    $db->query('INSERT INTO api_key (user_id, api_key, commercial, created, reason, estimated_usage) VALUES
+        (:user_id, :key, :commercial, NOW(), :reason, :estimated_usage)', array(
         ':user_id' => $THEUSER->user_id(),
         ':key' => $key,
         ':commercial' => $commercial,
-        ':reason' => $reason
+        ':reason' => $reason,
+        ':estimated_usage' => $estimated_usage
         ));
 }
 
 function api_key_form() {
 ?>
 <br>
-<h3>Apply for a new key</h3>
+<h3>Get a new key</h3>
 <form action="/api/key" method="post">
-<p>Is your application for:
-<input id="non_comm" type="radio" name="commercial" value="0"> <label for="non_comm">Non-commercial use</label>
-<input id="comm" type="radio" name="commercial" value="1"> <label for="comm">Commercial use</label>
+<p>About you:<br/>
+<input id="non_comm" type="radio" name="commercial" value="0"> <label for="non_comm">an individual pursuing a non-profit project on an unpaid basis</label><br/>
+<input id="non_comm_2" type="radio" name="commercial" value="0"> <label for="non_comm_2">a registered charity</label><br/>
```

Comment:
If we'd like to store these two options differently, I think you can just use a new value (presumably "2") to record it?